x/pocketcore/keeper/service.go

```diff
@@ -12,8 +12,13 @@ import (
 // HandleRelay handles an api (read/write) request to a non-native (external) blockchain
 func (k Keeper) HandleRelay(ctx sdk.Ctx, relay pc.Relay) (*pc.RelayResponse, sdk.Error) {
 	relayTimeStart := time.Now()
-	// get the latest session block height because this relay will correspond with the latest session
-	sessionBlockHeight := k.GetLatestSessionBlockHeight(ctx)
+
+	sessionBlockHeight := relay.Proof.SessionBlockHeight
```

Comment:
s/sessionBlockHeight/proofSessionHeight

Reply:
Context of this message?

Reply:
It's `sed` syntax for "suggestion to replace `sessionBlockHeight` with `proofSessionHeight` so its clearer/easier to read & understand.